CRM/CiviAwards/Hook/AlterAPIPermissions/Award.php

```diff
@@ -37,6 +37,8 @@ public function run($entity, $action, array &$params, array &$permissions) {
     ];
     $permissions['award_detail']['create'] = $permissions['award_detail']['update'] = $awardCreatePermission;
     $permissions['award_manager']['get'] = $awardCreatePermission;
+    $permissions['award_review_panel']['get'] = $awardCreatePermission;
+    $permissions['award_review_panel']['create'] = $permissions['award_review_panel']['update'] = $awardCreatePermission;
```

Comment:
it's better to have each permission assignation in its own line (like the `get` one above), so this is better:

```php
$permissions['award_review_panel']['get'] = $awardCreatePermission;
$permissions['award_review_panel']['create'] = $awardCreatePermission;
$permissions['award_review_panel']['update'] = $awardCreatePermission;
``